### DISCUSSION TITLE
disordered Vinplots under integrated analysis

### DISCUSSION THREAD

**Discussion by GreyRockIQ** [satijalab/seurat#5623] [Q&A]
Hello
I am having issues with vinplots of integrated analysis.
The labels appear thrice and the cluster number is disordered (picture attached).
![image](https://user-images.githubusercontent.com/96890415/153935367-2e4c5804-cdb5-4ed4-a0f3-6df3a05cdaa8.png)

Can someone help me to solve the issue?
Thanks
GreyRock

**Comment by timoast** [comment]
Can you provide the code you ran?

**Reply by GreyRockIQ** [reply]
Hi@Tim

Thanks for your response. 

Here is the code

plots <- VlnPlot(test_harmony, features = c("Hml"), split.by = "integration", group.by = "celltype",
    pt.size = 0, combine = FALSE)
wrap_plots(plots = plots, ncol = 1)


Thanks
GreyRock
